### PR TITLE
Debt company list

### PR DESCRIPTION
**Pull request by benbarnett**
* Adds list of companies who have shut down
* Adds item to FAQ
* Adds separate page listing companies who have shut down

List is deliberately surfaced twice for marketing reasons.

### CODE REVIEW

File: config/locales/debt_management/debt_management.cy.yml [hunk 1]
```diff
@@ -227,3 +230,42 @@ cy:
...
+      <p>Gallai rhai cwmnïau a oedd yn darparu Cynlluniau Rheoli Dyledion fod wedi gwerthu’u rhestr gleient i gwmni arall. I sicrhau bod y cwmni yr ydych chi’n cydweithio ag ef yn dal wedi ei awdurdodi gan yr FCA, gwiriwch y Gofrestr Credyd Defnyddwyr <a href="http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew"> http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew</a>.</p>
+    standalone_title: Cwmnïau nad ydynt yn gweinyddu Cynlluniau Rheoli Dyledion mwyach
+    standalone_intro: Mae’r FCA yn gwirio pob cwmni i sicrhau eu bod yn dilyn ei reolau ac yn trin eu cwsmeriaid yn deg. Mae’n gwneud hyn drwy ofyn iddynt wneud cais i gael eu hawdurdodi. Mae rhai cwmnïau ar y rhestr hon un ai wedi dewis peidio â gwneud cais neu heb gael eu hawdurdodi.
+    back_to_faq_html: Am ragor o wybodaeth darllenwch ein <a href="/cy/campaigns/debt-management/faq">Cwestiynau Cyffredin</a>
+    companies:
+      - text: Moneyfocus Financial Solutions Limited
```
Comment (asmega) [inline config/locales/debt_management/debt_management.cy.yml]: FYI I think you can make this as a list and not have the `text` key bit, http://yaml.org/spec/1.0/#id2489726

**Comment by asmega** [comment]
:+1:

File: config/locales/debt_management/debt_management.cy.yml (hunk 1 above)
Comment (benbarnett) [inline config/locales/debt_management/debt_management.cy.yml]: Will adjust

**Comment by darokel** [comment]
:+1:

File: config/locales/debt_management/debt_management.cy.yml [hunk 2]
```diff
@@ -227,3 +230,42 @@ cy:
...
+      <p>Gallai rhai cwmnïau a oedd yn darparu Cynlluniau Rheoli Dyledion fod wedi gwerthu’u rhestr gleient i gwmni arall. I sicrhau bod y cwmni yr ydych chi’n cydweithio ag ef yn dal wedi ei awdurdodi gan yr FCA, gwiriwch y Gofrestr Credyd Defnyddwyr <a href="http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew"> http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew</a>.</p>
+    standalone_title: Cwmnïau nad ydynt yn gweinyddu Cynlluniau Rheoli Dyledion mwyach
+    standalone_intro: Mae’r FCA yn gwirio pob cwmni i sicrhau eu bod yn dilyn ei reolau ac yn trin eu cwsmeriaid yn deg. Mae’n gwneud hyn drwy ofyn iddynt wneud cais i gael eu hawdurdodi. Mae rhai cwmnïau ar y rhestr hon un ai wedi dewis peidio â gwneud cais neu heb gael eu hawdurdodi.
+    back_to_faq_html: Am ragor o wybodaeth darllenwch ein <a href="/cy/campaigns/debt-management/faq">Cwestiynau Cyffredin</a>
+    companies:
+      - Moneyfocus Financial Solutions Limited
```
Comment (benbarnett) [inline config/locales/debt_management/debt_management.cy.yml]: @asmega is this what you meant?

File: app/controllers/debt_management_controller.rb [hunk 3]
```diff
@@ -7,6 +7,9 @@ def show
   def faq
   end
 
+  def companies
+  end
+
```
Comment (esbel) [inline app/controllers/debt_management_controller.rb]: As far as I remember, you don't need empty controller actions if the only thing they do is rendering a simple view. Correct me if I'm wrong, but I tried commenting out the other actions in this controller and it still did work.

File: config/locales/debt_management/debt_management.cy.yml (hunk 2 above)
Comment (asmega) [inline config/locales/debt_management/debt_management.cy.yml]: yeah :+1:

File: app/controllers/debt_management_controller.rb (hunk 3 above)
Comment (benbarnett) [inline app/controllers/debt_management_controller.rb]: Seems you're right – still works without these. I'll remove.

**Comment by esbel** [comment]
:ship::it:

**Pull request merged**